textstat/textstat.py

```diff
@@ -524,6 +525,19 @@ def text_standard(self, text, float_output=None):
                 upper_score, get_grade_suffix(upper_score)
             )
 
+    @repoze.lru.lru_cache(maxsize=128)
+    def reading_time(self, text, ms_per_char=14.69):
+        """
+        Function to calculate reading time (Demberg & Keller, 2008)
+        I/P - a text
+        O/P - reading time in millisecond
```

Comment:
Can we have an option to choose the output reading time units? I think that would make more sense and keep the default reading time in minutes?
thoughts @kishiyamat @alxwrd?

Reply:
@akarsh3007 Thank you for your suggestion! I agree with the idea that the reading time in milliseconds does not make sense. How many options do you think we need? I guess having three options(`second`, `minute`, and `hour`) is enough. In that case, the default value will be `minute`.

Reply:
I think that's perfect. The default should be minute.

Reply:
I think I would prefer not having the option to change the output as I feel it would dirty up the function call.

```python
reading_time("this is a test", output="minutes")
```

The `output` here feels weird.

---

I also think seconds should be returned as it will be the easiest for a user of the library to work with.

Milliseconds doesn't really provide much value, and minutes means we'll have to make a decision about how that looks. If the reading time is 2 and a half minutes, do we return `2.5` or `(2, 30)`?

So I think seconds would be best as it lets the user use that information how they please. This could also be a float, as a millisecond is a thousandth of a second.